docs/sources/whatsnew/whats-new-in-v10-1.md

```diff
@@ -59,3 +59,38 @@ The `use_refresh_token` configuration must be used in conjunction with the `acce
 
 The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.2.
 {{% /admonition %}}
+
+### OAuth role mapping enforcement
+
+<!-- AuthNZ -->
+
```

Comment:
Which editions of Grafana does this apply to: Enterprise, OSS, Cloud?

Reply:
All 3! I added more context at line 68.

Reply:
Okay I've moved and rewritten the note to be consistent with the rest of the What's new. We need the same information for the other two features as well.

Reply:
Changed!